cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -231,10 +231,28 @@ char* substr_utf8_int64(int64 context, const char* input, int32 in_len, int64 of
   return substr_utf8_int64_int64(context, input, in_len, offset64, in_len, out_len);
 }
 
+FORCE_INLINE
+char* concat_utf8_utf8(int64 context, const char* left, int32 left_len,
+                       bool left_validity, const char* right, int32 right_len,
+                       bool right_validity, bool* out_valid, int32* out_len) {
+  if (!left_validity) {
+    left_len = 0;
+  }
+  if (!right_validity) {
+    right_len = 0;
+  }
+  *out_valid = true;
```

Comment:
is this a null never function? since the output is set as always valid? (but function says null internal)..

can we please add a test case too.

Reply:
done